listenbrainz/listenstore/redis_listenstore.py

```diff
@@ -40,3 +40,12 @@ def check_connection(self):
         except redis.exceptions.ConnectionError as e:
             self.log.error("Redis ping didn't work: {}".format(str(e)))
             raise
+
+    def store_latest_listens(self):
+        """In dillema, because whether if I should put a cls from listen.py or not (cls is timestamp. Should I get this into the parameter?)
+        """
+        timestamp = None  # Haven't known where should I get the timestamp, so I left this out
+        data = Listen.from_json(timestamp).first()
+        latest_listens = data[:10]
+        top_ten_listens = self._redis.redis.setex("latest_listens:{}", latest_listens, 60)
+        return top_ten_listens
```

Comment:
there should be another function that returns the latest 10 listens

```
def get_latest_listens(self):
    return self.redis.get('latest_listens')
```

Reply:
I have applied it in my local (haven't pushed it yet to github)..
I'll apply the changes once I got home